jib-gradle-plugin/README.md

```diff
@@ -250,15 +257,27 @@ jib {
 
 You can add arbitrary, non-classpath files to the image by placing them in a `src/main/jib` directory. This will copy all files within the `jib` folder to the image's root directory, maintaining the same structure (e.g. if you have a text file at `src/main/jib/dir/hello.txt`, then your image will contain `/dir/hello.txt` after being built with Jib).
 
-You can configure a different directory by using the `extraDirectory` parameter in your `build.gradle`:
+You can configure a different directory by using the `extraDirectory` parameter in your `build.gradle`, either by using the `jib.extraDirectory.path` parameter:
```

Comment:
I think this would be clearer if the two options were first presented and then the examples below were shown with the two options, rather than separating the options descriptions with the examples.

Reply:
I wonder if it'd be clearer if, instead of talking about 1 way to configure 1 parameter + 2 ways to configure another parameter, we frame it as two possible options.

- a minimal configuration: 
```
extraDirectory = file('...')
```
- and a full configuration: 
```
extraDirectory {
  path = file('...')
  permissions = [...]
}
```

I think this would also help with the other ambiguities you and @chanseokoh mentioned.